FAQ.md

```diff
@@ -209,6 +227,39 @@ If we wanted to only warn for things that throw, then we would warn when calling
  
 NOTE(domenic): there is a conflict between static analysis and dynamic analysis. Consider the example let x : number = true ? 0 : false; A runtime type system would have no problem with this, whereas what you're proposing would disallow it. This is probably worth highlighting at a higher level in the document: it seems like you are leaning toward a statically-checkable model, which there is precedent for for JS but I believe other optionally-typed languages went a different direction.
 
+## Are you breaking the web?
+
+Nope. See [are all existing programs correct?](#are-all-existing-programs-correct).
+
+## Are all existing programs correct?
+
+NOTE(erights): specifically, are they all still statically valid? and if not, how do you avoid #breakingtheweb?
+
+[Interpreters](#terminology) remain unchanged, hence keeping all existing programs correct. For the newly introduced [typechecker](#terminology) some (previously valid) programs may be (desirably) invalidated.
+
+```javascript
+var x = 'foo'; // OK
+var y= x - 1; // OK
+// Typechecker Error, although Interpretation remains semantically and statically valid.
```

Comment:
Actually the typechecker should NOT error here, because x is not type annotated, and we are not standardizing inference.

We can only error in the `1 - 'true'` case, where no flowing of types is required.